src/pymatgen/util/testing/__init__.py

```diff
@@ -35,8 +35,15 @@
 FAKE_POTCAR_DIR = f"{VASP_IN_DIR}/fake_potcars"
 
 
-class PymatgenTest(TestCase):
-    """Extends unittest.TestCase with several assert methods for array and str comparison."""
+class MatSciTest:
```

Comment:
I originally didn't plan to deprecate `PymatgenTest` in this PR, but it turns out we have to do this here, otherwise tests that subclassing `PymatgenTest` would still rely on the `TestCase` and `pytest` methods like `setup_method/class` would not be compatible, meaning we have to do another migrate after `PymatgenTest` dropped `TestCase` inheritance. 

So I guess the easiest way is not to drop inheriting `TestCase` by `PymatgenTest`, but replacing `PymatgenTest` with another one that doesn't subclassing `TestCase`.

I drafted a name here but free feel to comment if you have better ideas (batch rename would be super easy with an IDE so don't worry).